docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md

```diff
@@ -1,638 +1,115 @@
 ---
 title: "Sorry, we don't have specifics on this C# error"
 ms.date: 08/04/2020
-f1_keywords: 
-  - "CS0002"
-  - "CS0003"
-  - "CS0004"
-  - "CS0008"
-  - "CS0012"
```

Comment:
This is one example where the error message is [already documented](https://github.com/dotnet/docs/blob/master/docs/csharp/misc/cs0012.md) but existed here. So, this f1 keyword was duplicated in this file and in the actual documentation.

Reply:
@BillWagner, while I was testing what Visual Studio will do in the case of duplicated f1 keywords in multiple articles, I noticed that **all error messages** goes to a Bing search after clicking F1. So, the f1_keywords metadata seems to be not working at all.

Reply:
Hi 👋 I dug into this problem before, see https://github.com/dotnet/roslyn/issues/40863 and https://github.com/dotnet/docs/issues/16448. I think the main sticking point was finding out what the new link should be for the f1 service - as I presume the msdn URL for is now deprecated.

If you did want to take it further, the NuGet repo has a good example, as it's set up to work for their error codes. Should be a fairly easy thing to implement, once someone can find out what that links meant to be! :smile:

Reply:
Hi 👋 I dug into this problem before, see https://github.com/dotnet/roslyn/issues/40863 and https://github.com/dotnet/docs/issues/16448. I think the main sticking point was finding out what the new link should be for the f1 service - as I presume the msdn URL for is now deprecated.

If you did want to take it further, the NuGet repo has a good example, as it's set up to work for their error codes. Should be a fairly easy thing to implement, once someone can find out what that links meant to be! :smile:

Reply:
@ChrisMaddock Thanks!! That's really really useful.
What I can't understand is, what's the problem to hook the error messages with the link `https://msdn.microsoft.com/query/dev16.query?appId=Dev16IDEF1&l=EN-US&k=k(ERROR_ID);k(DevLang-csharp)&rd=true.`.

Reply:
I'd presumed that link should be deprecated, given we've been replacing MSDN links everywhere else. There was a couple of other questions around filling in the appropriate appId and culture, but I assume that info is discoverable within Roslyn in some way. I think I was waiting on more info as to the official link we should be using here, and then other priorities came up, as they always do. 😄 

I'm not a Microsoft employee though - just a guy with a lot of compiler errors! So perhaps best to hear from the experts instead. :-)

Reply:
@ChrisMaddock The f1 for keywords in the latest visual studio still uses that msdn link. If you, for example, press f1 on an await keyword in the latest visual studio, it will open a similar msdn link.

Reply:
I opened https://github.com/dotnet/roslyn/pull/46850

Reply:
Great! 😄 Hope it gets merged soon!

Reply:
@ChrisMaddock I hope that too.
and many thanks for the information you provided, which is incredibly helpful.